src/index.js

```diff
@@ -51,10 +49,8 @@ async function getAutoBumpedVersion(baseTag) {
 
   logInfo(`Using ${tag} as base release tag for version bump`)
 
-  const { releaseType = 'patch' } = await conventionalRecommendedBumpAsync({
-    baseTag: tag,
-    config: conventionalCommitsConfig,
-  })
+  const bumper = new Bumper(process.cwd()).tag(baseTag)
+  const { releaseType = 'patch' } = await bumper.bump()
```

Comment:
this change very much depends on what the new API is. I would have imagined that the API had remained largely unchanged, but that doesn't seem to be the case if this requires this change